ARCMT.vr/description.ext

```diff
@@ -11,38 +11,6 @@ class CfgDebriefingSections {
     };
 };
 
-class CfgDebriefing
```

Comment:
This class is very rarely needed and if it is it can be added by the MM. On the opposite side MMs always need to look at this in order to go down to change their loadout settings.

Reply:
Returned it with only one custom ending so it does not take the whole page. This way MMs see it as an option without it taking too much space.